REQUIRE

```diff
@@ -1,6 +1,6 @@
 julia 0.3
-Nettle 0.2
-JSON 0.5
-ZMQ 0.3
-Compat 0.7.15
-Conda 0.1.5
+Compat 0.8.0
+Nettle 0.2.2
+JSON 0.5.2
+ZMQ 0.3.2
```

Comment:
0.5.2 isn't a good tag of JSON, it was accidentally breaking and later 0.5.x tags were actually earlier code